pkg/syncer/sync.go

```diff
@@ -200,3 +197,73 @@ func (s *Syncer) SyncWithChartsSyncer(ch *Chart, id, workdir, outdir string, has
 
 	return packagedChartPath, nil
 }
+
+func (s *Syncer) getMoverRequest(sourcePath, targetPath string) *mover.ChartMoveRequest {
+	req := &mover.ChartMoveRequest{
+		Source: mover.Source{
+			Chart: mover.ChartSpec{
+				Local: &mover.LocalChart{
+					Path: sourcePath,
+				},
+			},
+		},
+		Target: mover.Target{
+			Rules: mover.RewriteRules{
+				Registry:         s.target.GetContainerRegistry(),
+				RepositoryPrefix: s.target.GetContainerRepository(),
+				ForcePush:        true,
+			},
+			Chart: mover.ChartSpec{
+				Local: &mover.LocalChart{
+					Path: targetPath,
+				},
+			},
+		},
+	}
+	return req
+}
+
+func (s *Syncer) getBundleRequest(sourcePath, targetPath string) *mover.ChartMoveRequest {
```

Comment:
`get` makes it sounds like a generic getter, when in fact you want to get the request from the `get` action. There might be a better way to rename this, i.e

```
relok8sMoveReq
relok8sBundleSaveReq
relok8sBundleLoadReq
```